test/suite/test_hs18.py

```diff
@@ -531,6 +531,8 @@ def test_modifies(self):
         for i in range(0, 5):
             self.check_value(cursors[i], values[i])
 
+        self.session.checkpoint()
```

Comment:
Can you add a comment to explain the need for the checkpoint, so that it doesn't be removed later when this test is modified?

Reply:
Updated in [af2380a](https://github.com/wiredtiger/wiredtiger/pull/10473/commits/af2380a87b0d7447a11a5ee7dbfb4d1c9ca5e2f0)